### PR TITLE
feat ICKY 304 Added code for NuxtJs/Storefront UI App layout

### PR DESCRIPTION
**Pull request by kibo-nehabhanushali**
feat ICKY 304 Added code for NuxtJs/Storefront UI App layout @kibo-kevinwatts

### CODE REVIEW

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
There should be no dependencies on Vue-Storefront/core or Vue-Storefront/kibocommerce

File: components/AppHeader.vue [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  components: {
+    SfHeader,
+    SfImage,
+    LocaleSelector,
+    SfIcon,
+    SfButton,
+    HeaderNavigation,
+  },
+  directives: { clickOutside },
+  // setup(props, { root }) {
```
Comment (kibo-chandradeeptalaha) [inline components/AppHeader.vue:68]: Remove commented code

File: components/AppHeader.vue [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  setup() {
+    const isSearchOpen = ref(false)
+    const result = ref(null)
+
+    // const wishlistItemTotals = computed(() => {
```
Comment (kibo-chandradeeptalaha) [inline components/AppHeader.vue]: remove commented code from all files

File: package.json [hunk 3]
```diff
@@ -33,20 +33,27 @@
   "dependencies": {
     "@kibocommerce/graphql-client": "^1.0.2",
     "@nuxtjs/axios": "^5.13.6",
-    "@nuxtjs/composition-api": "^0.29.2",
+    "@nuxtjs/composition-api": "^0.29.3",
     "@nuxtjs/pwa": "^3.3.5",
+    "@nuxtjs/style-resources": "^1.2.1",
     "@storefront-ui/vue": "^0.11.0",
+    "@vue-storefront/core": "^2.4.3",
```
Comment (kibo-chandradeeptalaha) [inline package.json]: Remove these VSF dependencies.

File: nuxt.config.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { VSF_LOCALE_COOKIE } from "@vue-storefront/core"
```
Comment (kibo-chandradeeptalaha) [inline nuxt.config.js]: Why VSF is referenced here ?

File: components/AppHeader.vue [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+          </SfButton>
+        </div>
+      </template>
+    </SfHeader>
+  </div>
+</template>
+
+<script>
+import { SfHeader, SfImage, SfIcon, SfButton } from "@storefront-ui/vue"
+import { categoryGetters } from "@vue-storefront/kibocommerce"
```
Comment (kibo-chandradeeptalaha) [inline components/AppHeader.vue]: why VSF is referenced ?

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please check the provided comments and do the needful

**Comment by kibo-nehabhanushali** [comment]
> dependencies

@kibo-kevinwatts I have removed VSF dependency and commited code again in same PR , could you please take a look and let me know if anything else.

File: components/AppHeader.vue (hunk 1 above)
Comment (kibo-nehabhanushali) [reply components/AppHeader.vue:68]: done

File: components/AppHeader.vue (hunk 2 above)
Comment (kibo-nehabhanushali) [reply components/AppHeader.vue]: done

File: package.json (hunk 3 above)
Comment (kibo-nehabhanushali) [reply package.json]: done

File: nuxt.config.js (hunk 4 above)
Comment (kibo-nehabhanushali) [reply nuxt.config.js]: done

File: components/AppHeader.vue (hunk 5 above)
Comment (kibo-nehabhanushali) [reply components/AppHeader.vue]: done

**Comment by kibo-nehabhanushali** [comment]
> Please check the provided comments and do the needful

@kibo-chandradeeptalaha I have worked on all comments and pushed changes in same PR , could you please review

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**